tests-invoke

```diff
@@ -23,240 +23,81 @@ import socket
 import subprocess
 import sys
 
-sys.dont_write_bytecode = True
-
 from task import github
-from task import sink
 from machine.machine_core.directories import BASE_DIR
 
-
+sys.dont_write_bytecode = True
 HOSTNAME = socket.gethostname().split(".")[0]
-DEVNULL = open("/dev/null", "r+")
 
 
 def main():
     parser = argparse.ArgumentParser(description='Run integration tests')
-    parser.add_argument('--publish', dest='publish', default=os.environ.get("TEST_PUBLISH", ""),
-                        action='store', help='Publish results centrally to a sink')
     parser.add_argument('-v', '--verbose', action='store_true', help='Verbose output')
     parser.add_argument('--pull-number', help="The number of the pull request to test")
-    parser.add_argument('context', help="The context or type of integration tests to run")
+    parser.add_argument('--test-name', help='Test name',
+                        default=os.environ.get("TEST_NAME", "tests"))
+    parser.add_argument('--revision', help="Revision of the PR head",
+                        default=os.environ.get("TEST_REVISION"))
+    parser.add_argument('--context', help="The context or type of integration tests to run",
+                        default=os.environ.get("TEST_CONTEXT"))
```

Comment:
Oh, so was that the motivation for using env vars, so that they get shared between the logger and tests-invoke? Hmm. Does tests-invoke actually need all these? I would have thought it's mainly interested in setting `$TEST_SCENARIO`, which it parses from the context?

Reply:
well, it does need:
- `--pull-number` to detect collisions and thus needs also `--revision`. I am not sure where is the best place for this, but I guess here?
- `--test-name` seems indeed redundant as it is used only used to output name of the test
` `--context` is used as scenario